core/utils/token.ts

```diff
@@ -13,15 +13,14 @@ export function removeToken() {
 }
 
 export function getUserIdFromToken() {
-  return decodeToken()?.id;
+  return decodeToken()?.unique_name;
 }
 
 export function getTypeFromToken() {
```

Comment:
If you are returning role, why is it function called getTypeFromToken?

Reply:
The reason for that is that UserType is defined by our domain. ClaimTypes.Role represents the key that needs to be in the JWT payload for the [Authenticate] attribute to work.